sdk/core/azure-core/src/io/body_stream.cpp

```diff
@@ -82,35 +83,120 @@ int64_t MemoryBodyStream::OnRead(Context const& context, uint8_t* buffer, int64_
   return copy_length;
 }
 
-#if defined(AZ_PLATFORM_POSIX)
+int64_t FileBodyStream::GetFileSize(FILE* file)
```

Comment:
This should just call Win32 GetFileSizeEx or fstat

Reply:
Makes sense, will do that.